public/app/core/utils/markdown.ts

```diff
@@ -0,0 +1,19 @@
+import config from 'app/core/config';
+import marked from 'marked';
+
+const options: any = {
+  renderer: new marked.Renderer(),
+  pedantic: false,
+  gfm: true,
+  tables: true,
+  sanitize: !config.disableSanitizeHtml,
```

Comment:
@dprokop -- do you think this makes sense in `@grafana\runtime`?  

`@grafana\ui` would be better, but since it needs config it needs to be in runtime

Reply:
Yes, it does. I don't think grafana/ui would be better, as this is about data processing, and ui is about data display.